### ISSUE TITLE
Keep rotation center

### ISSUE THREAD

**Issue by SpooikypokDev** [dalboris/vpaint#70]
The rotation center should be kept between two operations on an object, as it would be more practical for rigging, etc.

**Comment by dalboris** [comment]
Yes that could be useful, though we will have to carefully think about how exactly to achieve this. Random thoughts to think about:

- Should they be preserved only during a given VPaint session, or across sessions (i.e., is it saved in the .vec file?). If it is for rigging, then across sessions of course.
- Are there use cases where preserving this rotation center would be undesirable/irritating? (I think yes)
- Should the behavior "preserve vs. non-preserve" be a user preference?
- Should the behavior "preserve during session vs. preserve across sessions" be a user preference?
- Should this choice of behavior be per-edge/face?
- What about a selection of multiple edges/faces?

At a higher-level, I feel that if the use case is for rigging, then maybe this functionality should be provided per-group/layers, once groups/layers are implemented. It would probably a better way to achieve the same functionality. Or or course, skeletons (#71) are the most natural choice to achieve this functionality.

**Comment by SpooikypokDev** [comment]
Yes, maybe you're right. I think #71 supersedes this. Shall we close this issue?

**Comment by dalboris** [comment]
No, let's leave it open for now, skeletons won't be implemented any time soon, and we need to some up with a reasonable solution for this functionality in the meantime.
